src/Dockerfile

```
@@ -0,0 +1,341 @@
+RUN apt update && apt -y install qemu-utils grub-pc parted udev fdisk e2fsprogs dosfstools kpartx cryptsetup-bin
+RUN apt -y install grub-efi  # this needs to be installed separately!
+
+ARG OUTPUTROOT=/output
+ARG OUTPUTDIR=${OUTPUTROOT}/rootfs
+
+# is required by passing build arg, value example: SP_VM_IMAGE_VERSION=build-1
+ARG SP_VM_IMAGE_VERSION
+ARG KERNEL_VERSION=6.12.13
+
+WORKDIR /output
+ADD kernel/files/scripts/log.sh /buildroot/files/scripts/
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)